nais-application/nais.yaml/reference.md

```diff
@@ -638,3 +638,21 @@ The lifetime in seconds for the issued refresh token from ID-porten. The refresh
 
 **Default**: `43200` \(12 hours\)
 
+## `spec.maskinporten`
+Configures a Maskinporten client for this application. See [Maskinporten](../security/auth/maskinporten.md) for more details.
+
+### `spec.maskinporten.enabled`
+If enabled, will provision a Maskinporten client for the application.
+
+**Default**: `false`
+
+### `spec.maskinporten.scopes[].scope`
+Valid list of Scopes that the Maskinporten client can be used for authentication and authorization, se the [Digdir maskinporten documentation for use of the JWT grant](https://difi.github.io/felleslosninger/maskinporten_protocol_token.html).
+
+Example: `[ "nav:some/scope" ]`
+
+{% hint style="info" %}
+Make sure that **NAV** have pre-registered rights to **all** the scopes `scopes`, specified in the manifest, or provision of client will fail.
```

Comment:
Already mentioned in the scopes section in the link below. Should just refer to it instead of repeating the info here.